cliconfig/config.go

```diff
@@ -117,49 +186,17 @@ func Load(configDir string) (*ConfigFile, error) {
 	if _, err := os.Stat(confFile); err != nil {
 		return &configFile, nil //missing file is not an error
 	}
-
-	b, err := ioutil.ReadFile(confFile)
+	file, err := os.Open(confFile)
```

Comment:
we need a `defer file.Close()` after checking the error returned here. Otherwise we won't close the file properly.